README.md

```diff
@@ -79,6 +79,34 @@ For example:
 It will display a table view with latest packets collected and write data under output/pcap directory.
 To stop capturing press Ctrl-C.
 
+### FLow Capture and write to Sqlite database
+
+```bash
+./oc/oc-netobserv-flows-db
+```
+
+This will write flows to `flows.db` file and it can be inspected using `sqlite3` for example 
+
+```bash
+$ oc exec -it -n netobserv-cli collector -- bash
+bash-5.1$ sqlite3 flows.db 
```

Comment:
Currently, flows.db is in /tmp folder

In the future, I would like to have it in /output folder as other export files.
That allows `cleanup` function to copy all output to your local machine before deleting namespace

Reply:
moved